DogeNet.BLL/Features/Conversations/CreateConversation/CreateConversationHandler.cs

```diff
@@ -0,0 +1,54 @@
+﻿// <copyright file="CreateConversationHandler.cs" company="Leobit">
+// Copyright (c) Leobit. All rights reserved.
+// </copyright>
+
+namespace DogeNet.BLL.Features.Conversations.CreateConversation
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using AutoMapper;
+    using DogeNet.BLL.Services.Interfaces;
+    using DogeNet.DAL;
+    using DogeNet.DAL.Models;
+    using MediatR;
+    using Microsoft.AspNetCore.Identity;
+
+    public class CreateConversationHandler : IRequestHandler<CreateConversationCommand>
+    {
+        private readonly DBContext context;
+
+        private readonly IMapper mapper;
+
+        public CreateConversationHandler(DBContext context, IMapper mapper)
+        {
+            this.context = context;
+            this.mapper = mapper;
+        }
+
+        public async Task<Unit> Handle(CreateConversationCommand request, CancellationToken cancellationToken)
+        {
+            var conversation = this.mapper.Map<Conversation>(request.model);
+            conversation.CreatedAt = System.DateTime.Now;
+
+            await this.context.Conversations.AddAsync(conversation);
+
+            var sender = new ConversationParticipant()
```

Comment:
Should the conversation participant include some reference to the conversation entity?

Reply:
it is not mandatory, when I add references here
![image](https://user-images.githubusercontent.com/77879337/145020571-3300c9ef-f9a9-42e6-b1f6-4cd1f13fb03c.png)

Reply:
![image](https://user-images.githubusercontent.com/77879337/145020665-c401fca2-e9a2-4fa2-b14e-a964b76df1c6.png)
As you can see every participant have reference to user table and to conv table